Automattic/jetpack
# Blaze: add new Tracks even to track panel views.

## Proposed changes:

It will allow us to build more accurate funnels for that flow.
See p1673443048626029-slack-C04DZ725FEK

### Other information:


## Jetpack product discussion

* 1386-gh-dotcom-forge

## Does this pull request change what data or activity we track or use?

* Yes, it tracks views of the post-publish panel every time you publish a post on your site.

## Testing instructions:

On a WoA site or WordPress.com simple site:

* Go to Posts > Add New
* Open the JavaScript console
* Type in `localStorage.setItem( 'debug', 'dops:analytics*' );`
* Check the box to preserve logs
* Refresh the page and keep your JavaScript console open.
* Write a post and hit publish.
* You should see the panel in the post-publish sidebar.
* You should also see a line in your console showing the Tracks event
* Click on the button, and you should see another Tracks event for the click.
* Now that you've checked that it works, you should check when it should **not** work: when editing posts that are already published, for example.

<img width="899" alt="Screenshot 2023-01-16 at 18 43 24" src="https://user-images.githubusercontent.com/426388/212739550-36f96808-d59d-4ffa-9d60-79edef441735.png">
